app/src/main/java/com/cornellappdev/coffee_chats_android/SchedulingActivity.kt

```diff
@@ -0,0 +1,101 @@
+package com.cornellappdev.coffee_chats_android
+
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+import androidx.constraintlayout.widget.ConstraintLayout
+import kotlinx.android.synthetic.main.activity_scheduling.*
+import android.widget.AdapterView.OnItemClickListener
+import kotlinx.android.synthetic.main.activity_scheduling.back_button
+import kotlinx.android.synthetic.main.fragment_create_profile.*
+
+
+class SchedulingActivity : AppCompatActivity() {
+    var currDay: String = "Sunday"
+    lateinit var currDayTextView: TextView
+    private val days = arrayOf("Su", "M", "Tu", "W", "Th", "F", "Sa")
+    private val daysFullName = arrayOf("Sunday", "Monday", "Tuesday", "Wednesday", "Thursday",
+        "Friday", "Saturday")
+    private val times = arrayOf("9:00", "1:00", "5:00", "9:30", "1:30", "5:30", "10:00", "2:00",
+    "6:00", "10:30", "2:30", "6:30", "11:00", "3:00", "7:00", "11:30", "3:30", "7:30", "12:00",
+    "4:00", "8:00", "12:30", "4:30", "8:30")
```

Comment:
is this data for debugging?

Reply:
No these are the actual time (meeting times) we display on screen